### ISSUE TITLE
Missing documentation regarding bundled parsers

### ISSUE THREAD

**Issue by MariaSolOs** [nvim-treesitter/nvim-treesitter#5390]
Neovim is now bundling some parsers (those included [here](https://github.com/neovim/neovim/tree/1f551e068f728ff38bd7fdcfa3a6daf362bab9da/runtime/queries)) and `nvim-treesitter`'s README is unclear regarding what to do with this. The docs indicate that one shouldn't have more than one parser runtime directory, yet this will be the case when including parsers not included by the editor.

> "If you get more than one path, remove the ones that are outside this plugin (nvim-treesitter directory), so the correct version of the parser is used."

I imagine one should not try to try to remove the parsers inside `lib/nvim/parser`. And so should we remove these from `ensure_installed`?

**Comment by amaanq** [comment]
Yeah I guess that could be updated to say "more than one path that's not where the bundled neovim parsers are"

You'd want to just install those parsers with nvim-treesitter as well iirc

**Comment by MariaSolOs** [comment]
@amaanq right, but given that `nvim-treesitter` is one of the first plugins most Neovim newcomers will install, I believe the docs should be clearer about this :)

**Comment by amaanq** [comment]
Feel free to submit a PR! :grin:

**Comment by MariaSolOs** [comment]
@amaanq I'll be glad to once I know what the right instructions are :)

**Comment by amaanq** [comment]
Just edit the readme reflecting what i said more or less - if your neovim installation has bundled parsers, install them in nvim-treesitter as well to avoid breakage, then edit the bit where if you have more than one parser runtime directory that is *not* the one where neovim's parsers are, remove those

@clason might be better suited at explaining this

**Comment by clason** [comment]
> is one of the first plugins most Neovim newcomers will install

but they shouldn't, is my point -- not while it's still experimental. The README already explains how to handle the bundled parsers on stable, and newcomers should **not** be using nightly.

**Comment by clason** [comment]
That being said, I've had enough complaints to bite the bullet and rewrite the legacy README, even at the cost of more rebasing work for me.
